tests/vector-tests.lisp

```diff
@@ -0,0 +1,54 @@
+;;;; tests/vector-tests.lisp
+;;;;
+;;;;
+
+(in-package #:magicl-tests)
+
+;; MAGICL:= only specializes on tensor types, but we need an analog to numpy.isclose for scalars
+(defgeneric == (val1 val2 &optional epsilon)
```

Comment:
I think it might be good to have this as part of the high-level module. This could even be specializations on `magicl:=`

Reply:
I think that makes sense; I just didn't want to start adding stuff willy-nilly to the main project. :)